src/assets/svg/Icons/NotificationBellSVG.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { NotificationProps } from '../../../types';
+
+const NotificationBellSVG: React.FC<NotificationProps> = ({
```

Comment:
```suggestion
const NotificationBell: React.FC<NotificationProps> = ({
```
Rename the component folder as well `NotificationBell`